Tests/Xcodeproj/FunctionalTests.swift

```diff
@@ -21,13 +21,55 @@ class FunctionalTests: XCTestCase {
             XCTAssertXcodeprojGen(prefix)
             let pbx = Path.join(prefix, "Library.xcodeproj")
             XCTAssertDirectoryExists(pbx)
-            // FIXME: Move this to a method.
-            try popen(["env", "-u", "TOOLCHAINS", "xcodebuild", "-project", pbx])
+            XCTAssertXcodeBuild(project: pbx)
```

Comment:
the fixture is copied into a temp folder somewhere `/var/.../something/`
xcodebuild builds in a folder called "build" inside the copied fixture